torch/testing/_internal/common_utils.py

```diff
@@ -623,7 +623,7 @@ def is_avx512_vnni_supported():
         return False
     with open("/proc/cpuinfo", encoding="ascii") as f:
         lines = f.read()
-    return "avx512vnni" in lines
+    return any(s in lines for s in ("avx512_vnni", "avx512vnni"))
```

Comment:
Wouldn't that accomplish the same?
```suggestion
    return "vnni" in lines
```

Reply:
It will, it's just less strict. I'm not guessing we get any non-AVX512 VNNI instructions in the future - but if we would, this would match those as well. Then again, since the only consequence of an unintended match is that a single test in the test suite is skipped, it doesn't seem too bad. I've changed it.